src/idl_parser.cpp

```diff
@@ -1314,6 +1350,38 @@ CheckedError Parser::ParseSingleValue(Value &e) {
   return NoError();
 }
 
+void Parser::InitializeNestedFlatbufferParser(Parser& nestedParser, Value *const nested) {
+  assert(nested);
```

Comment:
Then they would copy the same things from the parent, so that should be ok? Frankly recursive nested flatbuffers sounds a bit crazy :) I'd say keep the code as simple as possible for now.